learn/advanced/asynchronous_operations.md

```diff
@@ -32,75 +32,93 @@ All of Meilisearch's asynchronous operations belong to a category called "tasks"
 
 All asynchronous operations return a summarized version of [the full `task` object](/reference/api/tasks.md#task-object). It contains the following fields in the stated order:
 
-| Field      | Type    | Description                              |
-|------------|---------|---------------------------------         |
-| `taskUid`        | integer | Unique sequential identifier             |
-| `indexUid`   | string  | Unique index identifier (always `null` for dumps)                  |
-| `status`     | string  | Status of the task. Value is `enqueued`  |
-| `type`       | string  | Type of task                             |
-| `enqueuedAt` | string  | Represents the date and time in the RFC 3339 format when the task has been `enqueued`                                                        |
+| Field            | Type    | Description                                                                           |
+| :--------------- | :------ | :------------------------------------------------------------------------------------ |
+| **`taskUid`**    | Integer | Unique sequential identifier                                                          |
+| **`indexUid`**   | String  | Unique index identifier (always `null` for [global tasks](#global-tasks))             |
+| **`status`**     | String  | Status of the task. Value is `enqueued`                                               |
+| **`type`**       | String  | Type of task                                                                          |
+| **`enqueuedAt`** | String  | Represents the date and time in the RFC 3339 format when the task has been `enqueued` |
 
 You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks.md#get-one-task).
 
+### Global tasks
+
+Some task types are not associated with a particular index but apply to the entire instance. These tasks are called global tasks. Global tasks display `null` for the `indexUid` field.
+
+Meilisearch considers the following task types as global:
```

Comment:
Maybe we should also note here that, in order to **read/access** these tasks in the task queue, your API key must give you the right to read the tasks on all indexes?